Sources/TuistSigning/SigningCipher.swift

```diff
@@ -0,0 +1,134 @@
+import Basic
+import CryptoSwift
+import Foundation
+import TuistCore
+import TuistSupport
+
+enum SigningCipherError: FatalError, Equatable {
+    case failedToEncrypt
+    case masterKeyNotFound(AbsolutePath)
+    case rootDirectoryNotFound(AbsolutePath)
+
+    var type: ErrorType { .abort }
+
+    var description: String {
+        switch self {
+        case .failedToEncrypt:
+            return "Encryption failed"
```

Comment:
Can we provide more information to the end-user about why the encryption failed? This might help when debugging issues related to signing.

Reply:
Agree. We should make it easier for users to understand what's happening and for us to help them debug the issues.

Reply:
I have tried my best to describe what might have gone wrong